blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -25,43 +25,98 @@ The ``OVER`` clause defines the ``window`` containing the appropriate rows
+the rows are considered a single partition.
 
+Example::
 
-If ``ORDER BY`` is supplied the ``window`` definition consists of a range of
-rows starting with the first row in the result set and ending with the current
-row, plus any subsequent rows that are equal to the current row, which are the
-current row's ``peers``.
+   cr> select dept_id, row_number() OVER(PARTITION BY dept_id) from employees order by 1, 2;
+   +---------+------------------------------------------+
+   | dept_id | row_number() OVER (PARTITION BY dept_id) |
+   +---------+------------------------------------------+
+   |    4001 |                                        1 |
+   |    4001 |                                        2 |
+   |    4001 |                                        3 |
+   |    4002 |                                        1 |
+   |    4002 |                                        2 |
+   |    4002 |                                        3 |
+   |    4002 |                                        4 |
+   |    4003 |                                        1 |
+   |    4003 |                                        2 |
+   |    4003 |                                        3 |
+   |    4003 |                                        4 |
+   |    4003 |                                        5 |
+   |    4004 |                                        1 |
+   |    4004 |                                        2 |
+   |    4004 |                                        3 |
+   |    4006 |                                        1 |
+   |    4006 |                                        2 |
+   |    4006 |                                        3 |
+   +---------+------------------------------------------+
+   SELECT 18 rows in set (... sec)
 
+If ``ORDER BY`` is supplied the ``window`` definition consists of a range of
+rows starting with the first row in the ``partition`` and ending with the
+current row, plus any subsequent rows that are equal to the current row, which
+are the current row's ``peers``.
 
 Example::
 
-   cr> select col1, sum(col1) over (order by col1) from unnest([1, 5, 2, 3, 2, 5, 4]);
-   +------+--------------------------------------+
-   | col1 | sum(col1) OVER (ORDER BY "col1" ASC) |
-   +------+--------------------------------------+
-   |    1 |                                    1 |
-   |    2 |                                    5 |
-   |    2 |                                    5 |
-   |    3 |                                    8 |
-   |    4 |                                   12 |
-   |    5 |                                   22 |
-   |    5 |                                   22 |
-   +------+--------------------------------------+
-   SELECT 7 rows in set (... sec)
+   cr> select dept_id, sex, count(*) OVER(PARTITION BY dept_id ORDER BY sex) from employees order by 1,2,3;
+   +---------+-----+--------------------------------------------------------+
+   | dept_id | sex | count(*) OVER (PARTITION BY dept_idORDER BY "sex" ASC) |
```

Comment:
do we have a formatter issue here that there is no space between `dept_id` and `ORDER BY` ?